src/app/workflow/register-workflow-modal/register-workflow-modal.component.ts

```diff
@@ -134,6 +135,7 @@ export class RegisterWorkflowModalComponent implements OnInit, AfterViewChecked,
       }
     });
     this.selectInitialSourceControlRepository();
+    this.workflowPathError = validationMessages.cwlPath.pattern;
```

Comment:
What is this for? Seems wrong off-hand to pick a default that may not match the descriptor type. Can't you just leave it null?

If you can leave it null, then you should change line 57 to `public workflowPathError: String | null;`

Reply:
Actually the line ended up being unnecessary. Removed